File: fundamentals/arrayAccess.js
export const execute = (index) => {
  const arr = [1, 2, 3, 4, 5];

  console.log(`Array: [${arr.join(', ')}]`);
  console.log(`Accessing index \x1b[33m${index}\x1b[0m`);
  console.log(`Value: \x1b[32m${arr[index]}\x1b[0m\n`);

  return arr[index];
};

File: fundamentals/array_access.py
def execute(index):
    arr = [1, 2, 3, 4, 5]
    print(f"Array: {arr}")
    print(f"Accessing index \033[33m{index}\033[0m")
    print(f"Value: \033[32m{arr[index]}\033[0m\n")

    return arr[index]